apps/smart-contracts/core/test/RedeemHook.test.ts

```diff
@@ -102,4 +111,75 @@ describe('=> RedeemHook', () => {
       await redeemHook.connect(deployer).setTokenSender(tokenSender.address)
     })
   })
+
+  describe('hook', () => {
```

Comment:
nit: (same for others)
```suggestion
  describe('# hook', () => {
```